GOVERNANCE.md

```diff
@@ -57,7 +57,18 @@ If the PR author is a maintainer or an owner of given component, this counts as
 
 Proposals can be opened against the [proposals repository](https://github.com/strimzi/proposals).
 Proposals should cover any changes to the Strimzi project which might significantly impact its users or the project direction.
-Proposals are approved by a ⅔ majority maintainers vote.
+
+Voting about proposals is using +1, 0, -1 votes and their fractions, where:
+* +1 means "yes"
+* -1 means "no"
+* The numbers between +1 and -1 indicate how strongly you feel about the proposal
```

Comment:
I've never seen a +0.5 used in practice. So I wonder if we really need this.

Reply:
I agree it is not used that often. But I think it is a nice way to express your opinion if you don't feel like giving a +1 or -1. I would prefer to keep this.